src/components/ChatLog.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types'
+import ChatEntry from './ChatEntry.js';
+
+function ChatLog( { updateChatLog, entries } ){
+    const chatComponents = [];
+    const updateChatLogProp = updateChatLog
+
+    for (const chat of entries) {
+        chatComponents.push(
+            <li key={chat.id}>
```

Comment:
Since this is a list item, dots appear on the screen for each ChatEntry on the webpage. To make the dots go away, either do not use the `li` element or update the CSS to make it so the dots do not appear for the `li` element. Also, all of these `li` elements are under a `div` element, which should not happen. The `li` element should be under `ul`, `ol`, or `menu`.